### PR TITLE
data: Akshaykk12

### PR DESCRIPTION
**Pull request by Akshaykk12**

### CODE REVIEW

**Review by imbeshat — DISMISSED** [review]

File: data/Akshaykk12.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/akshay-kudalkar/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/Akshaykk12",
+      "icon": "FaGit"
```
Comment (priyankarpal) [inline data/Akshaykk12.json:24]: Just change the icon from `FaGit` to `FaGithub`
``` suggestion
      "icon": "FaGithub"
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by himanshu1221 — CHANGES_REQUESTED** [review]
Hey @Akshaykk12 please Fix your `GitHub` icon

**Review by dev-rakeshshaw — DISMISSED** [review]
LGTM👍

**Review by iamthecloverly — DISMISSED** [review]
LGTM

File: data/Akshaykk12.json (hunk 1 above)
Comment (Akshaykk12) [reply data/Akshaykk12.json:24]: Thank you

**Comment by Akshaykk12** [comment]
> LGTM

👍

**Comment by Akshaykk12** [comment]
Is it all good to go?

**Comment by dev-rakeshshaw** [comment]
> Is it all good to go?

Yes...now wait for someone to merge your PR.

**Review by himanshu1221 — APPROVED** [review]
**LGTM :+1:**

**Thanks :tada:**

**Review by mahfuztbt — APPROVED** [review]

**Review by amandamartin-dev — APPROVED** [review]

**Pull request merged**